pkg/localstorev2/internal/upload/uploadstore_test.go

```diff
@@ -41,15 +41,7 @@ var _ internal.Storage = (*testStorage)(nil)
 type testStorage struct {
 	ctx        context.Context
 	indexStore storage.Store
-	chunkStore *chunkStore
```

Comment:
Actually, I got confused because this testStorage is no longer used I think. All the tests now use the common function in the `internal.go` file. So this can be removed completely.

Reply:
I have added GetWithStamp method to localstore/ChunkStore. I am not sure what do about other ting you are suggesting here?

Reply:
So you can remove the testStorage object completely I think. Its not being used. So remove L41-L49